runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -335,4 +347,38 @@ public void onVarbitChanged(VarbitChanged event)
 			trawlerNotificationSent = false;
 		}
 	}
+
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.FISHING_TRAWLER_GROUP_ID)
+		{
+			startTime = Instant.now();
```

Comment:
Won't this show you the wrong time if you enable the plugin while midway through a game?

Reply:
Just tested to make sure... The plugin works correctly even when turned on midway through a game. startTime is initialised when the game loads, regardless of the plugin being enabled.

Reply:
That is not true (unless you mean just the boolean config fir fishing trawler)

Reply:
Yes that's what I mean. Sorry if my wording was confusing

Reply:
So this basically would not work properly if the plugin was enabled while in the middle of a trawler trip? Seems like that could be added, though if it would be a lot of code just to account for that edge case, it can probably be left as-is.

Reply:
If "Fishing" is enabled mid game the timer will be incorrect. I don't think we should add complexity to resolve this though.

Reply:
Just wanted to tidy up and close some of my pull request before starting something new. Is this ok to go through? @Nightfirecat is correct and this wont work if enabled mid-way through a trawler trip, but I dont see this as a problem